content/bn/containerization.md

```diff
@@ -0,0 +1,18 @@
+---
+title: কন্টেইনারাইজেশন
+status: Completed
+category: প্রযুক্তি
+---
+
+## এটা কি
+
+কন্টেইনারাইজেশন হল একটি প্রক্রিয়া যা একটি অ্যাপ্লিকেশন এবং এর সংশ্লিষ্ট জিনিসসমূহকে একটি [কন্টেইনার ইমেজ](/container-image/) বান্ডিল করার প্রক্রিয়া। কন্টেইনার নির্মাণ প্রক্রিয়ার জন্য [ওপেন কন্টেইনার ইনিশিয়েটিভ](https://opencontainers.org) (OCI) মান মেনে চলা প্রয়োজন। যতক্ষণ না আউটপুট একটি কন্টেইনার ইমেজ যা এই স্ট্যান্ডার্ড মেনে চলে, কোন কন্টেইনারাইজেশন টুল ব্যবহার করা হয় তা বিবেচ্য নয়।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+কনটেইনারগুলি প্রচলিত হওয়ার আগে, সংস্থাগুলি যেকোনো  [বেয়ার-মেটাল মেশিন](/bare_metal_machine/) এর একাধিক অ্যাপ্লিকেশন তৈরি করার জন্য ভার্চুয়াল মেশিনের (VM) উপর নির্ভর করত। ভিএমগুলি পাত্রের তুলনায় উল্লেখযোগ্যভাবে  বড় এবং যার ফলে এটি চালানোর জন্য একটি হাইপারভাইজার প্রয়োজন।যেহেতু এই বৃহৎ ভিএম টেমপ্লেটগুলির স্টোরেজ, ব্যাকআপ এবং স্থানান্তরের কাজ করে, এ কারণে ভিএম টেমপ্লেট তৈরি করাও একটি ধীর প্রক্রিয়া। অতিরিক্তভাবে, ভিএমগুলি যদি  [অপরিবর্তনশীলতা](/ অপরিবর্তনীয়_ইনফ্রাস্ট্রাকচার/) নীতি লঙ্ঘন করে, তবে এটি কনফিগারেশন ড্রিফটে ভুগতে পারে। 
```

Comment:
It seems that the link has been localized `[অপরিবর্তনশীলতা](/ অপরিবর্তনীয়_ইনফ্রাস্ট্রাকচার/)` thus this link is not working